xml/http_servers.xml

```diff
@@ -3510,4 +3510,29 @@
       <param pos="0" name="service.product" value="gSOAP"/>
       <param pos="1" name="service.version"/>
     </fingerprint>
+
+           <!-- Apple QuickTime streaming server -->
+    <fingerprint pattern="^QTSS\/([\d*.]*) \(Build\/[\d*.]*; Platform\/MacOSX; Release\/Panther">
+      <description>QTSS on OS X 10.3</description>
+      <example service.version="5.0">QTSS/5.0 (Build/452; Platform/MacOSX; Release/Panther; )</example>
+      <example service.version="5.0.3.3">QTSS/5.0.3.3 (Build/452.22.4; Platform/MacOSX; Release/Panther; Update/Security; )</example>
+      <param pos="0" name="os.vendor" value="Apple"/>
+      <param pos="0" name="os.family" value="Mac OS X"/>
+      <param pos="0" name="os.product" value="Mac OS X"/>
+      <param pos="0" name="os.device" value="General"/>
+      <param pos="0" name="os.version" value="10.3"/>
+      <param pos="0" name="service.product" value="QTSS"/>
+      <param pos="1" name="service.version"/>
+    </fingerprint>
+
+    <fingerprint pattern="^QTSS\/([\d*.]*) \(Build\/[\d*.]*; Platform\/MacOSX; Release\/Mac OS X">
```

Comment:
Here and the other places where you are matching version-like strings, I think `[\d*.]*` could be improved -- the `*` inside will match the literal *, the `.` should be explicitly escaped and the  `*` at the end should probably be a `+`.  I'd suggest `[\d\.]+`.